### DISCUSSION TITLE
Can't fix difference between APP_ENV:dev and :prod

### DISCUSSION THREAD

**Discussion by phlpnmsr** [pimcore/pimcore#11385] [General]
Hi, I'm totally new to Pimcore and actually not a developer, nevertheless I got Pimcore installed on Google Cloud VM via Click-to-Deploy. I connect to GCP via Atom/ssh and upload modifications of templates and static files (css). So far so good. When in APP_ENV:dev mode it works fine and I see the changes. But once I switch the var to APP_ENV:prod, some old status of the modified files appears - and I do not have a clue why. Clearing caches via the admin backend (all symfony and data) doesn't help either.

In the documentation I could not find anything hinting to the solution. Then I thought maybe it needs some kind of deployment to publish the dev files to prod but also here couldn't find anything. In case I did not read the docs properly apologies upfront ...

**Comment by dpfaffenbauer** [comment]
you need to cache clear in prod if you do changes:

```
bin/console cache:clear
```

**Reply by phlpnmsr** [reply]
@dpfaffenbauer  thx a lot!
done it!
<img width="664" alt="Bildschirmfoto 2022-02-07 um 15 09 41" src="https://user-images.githubusercontent.com/45699271/152804351-98580709-e08a-4528-8c36-49fbb1953f0a.png">

now it's:
<img width="218" alt="Bildschirmfoto 2022-02-07 um 15 12 55" src="https://user-images.githubusercontent.com/45699271/152804375-48f34164-5602-4aa6-be5c-0324f617c061.png">

**Reply by dpfaffenbauer** [reply]
then you messed up somewhere else in your templates or so

**Reply by phlpnmsr** [reply]
hm, I'm afraid it is so - after clearing once more I get a 500 ..
Just a last one: is there any docu on which rights the folders and files should ideally have in the project, like for the folders public, var, templates ... maybe it's related to that.
<img width="491" alt="Bildschirmfoto 2022-02-07 um 17 54 47" src="https://user-images.githubusercontent.com/45699271/152834800-a7a9a36b-93ab-4027-bf11-31e648dd29fc.png">

**Reply by phlpnmsr** [reply]
fixed it by manually removing /opt/pimcore/demo/var/cache/
